src/ophyd_async/fastcs/eiger/_eiger_controller.py

```diff
@@ -47,20 +43,20 @@ async def prepare(self, trigger_info: TriggerInfo):
             self._drv.trigger_mode.set(
                 EIGER_TRIGGER_MODE_MAP[trigger_info.trigger].value
             ),
-            self._drv.num_images.set(trigger_info.total_number_of_triggers),
+            self._drv.nimages.set(trigger_info.total_number_of_triggers),
         ]
         if trigger_info.livetime is not None:
             coros.extend(
                 [
-                    self._drv.acquire_time.set(trigger_info.livetime),
-                    self._drv.acquire_period.set(trigger_info.livetime),
+                    self._drv.count_time.set(trigger_info.livetime),
+                    self._drv.frame_time.set(trigger_info.livetime),
                 ]
             )
         await asyncio.gather(*coros)
 
     async def arm(self):
         # TODO: Detector state should be an enum see https://github.com/DiamondLightSource/eiger-fastcs/issues/43
-        self._arm_status = await set_and_wait_for_other_value(
+        self._arm_status = set_and_wait_for_other_value(
```

Comment:
We still need the await here:
https://blueskyproject.io/ophyd-async/main/how-to/interact-with-signals.html#wait-for-the-value-to-match-some-expected-value
```suggestion
        self._arm_status = await set_and_wait_for_other_value(
```